cmd/stack.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/elastic/elastic-package/internal/stack"
 )
 
+var availableServices = map[string]bool{
+	"elasticsearch":    true,
+	"kibana":           true,
+	"package-registry": true,
```

Comment:
Nit: Neat trick I learnt from Steffen: for such "existence lookup" maps, we can use `map[string]struct{}` as `struct{}` takes up 0 bytes whereas `bool` takes up 1 byte 😅.

Reply:
You're right, let's switch to this :)